internal/cmd/gocdk/provision.go

```diff
@@ -54,12 +60,170 @@ func registerProvisionCmd(ctx context.Context, pctx *processContext, rootCmd *co
 	rootCmd.AddCommand(provisionCmd)
 }
 
+const provisionStaticRootDir = "/provision"
+
 // The "provision list" command.
-func provisionList(pctx *processContext) {
-	pctx.Logf("NOT YET IMPLEMENTED")
+func provisionList(pctx *processContext) error {
+	// TODO(rvangent): There must be an easier way to get a listing... add
+	// a helper to static.
+	dir, err := static.Open(provisionStaticRootDir)
+	if err != nil {
+		return xerrors.Errorf("provision list: couldn't open static resources: %w", err)
+	}
+	ptInfos, err := dir.Readdir(-1)
+	if err != nil {
+		return xerrors.Errorf("provision list: couldn't read static resources: %w", err)
+	}
+	// The entries at this level are portable APIs, e.g., "blob".
+	for _, ptInfo := range ptInfos {
```

Comment:
We likely want sort this list. I'm not sure if `Readdir`'s order is guaranteed.

Reply:
It is "in directory order", which is at least stable, so I think we're OK.
https://golang.org/pkg/os/#File.Readdir